README.md

```diff
@@ -57,7 +58,22 @@ There are some issues with the `.off` file format. Use e.g. meshlab to convert t
 sudo apt-get install libxrandr-dev
 sudo apt-get install libxinerama-dev
 ```
-List of potential libs not found: `libxrandr`, `libxinerama`, `libxcursor`, `libxi`
+
+- if `libboost` not found install all related packages via
+```bash
+sudo apt-get install libboost-all-dev
+```
+
+List of potential libs not found: `libxrandr`, `libxinerama`, `libxcursor`, `libxi`, `libboost`
+
+### CMake Error
```

Comment:
please remove the cmake error as it is fixed now :)

Reply:
Sure. Done.